### PR TITLE
helm: Add web deployment

### PR DESCRIPTION
**Pull request by ecerulm**
This PR : 
* add the web UI ( deployment, service and ingress) with `web.enabled=true`
* fixes issues with missing environment variables in the api container which leads to api starting in random ports 
* wrong configmap name 
* wrong replicaCount

closes: #1016 
closes: #1015

### CODE REVIEW

File: chart/templates/deployment-web.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+              port: 3000
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 3000
+          env:
+            - name: MARQUEZ_PORT
+              value: {{ .Values.marquez.service.port | quote }}
+            - name: MARQUEZ_HOST
+              value: {{ include "marquez.fullname" . }}-api
```
Comment (KevinMellott91) [inline chart/templates/deployment-web.yaml:46]: I like this approach for MARQUEZ_HOST much better to what I had here (another chart value setting)

**Comment by KevinMellott91** [comment]
@ecerulm I've reviewed your PRs, will you please do the same to mine (PR #1019) and let me know your thoughts? Here is how I think our work can come together nicely - please let me know what you think.

My PR introduces the [Bitnami common chart](https://github.com/bitnami/charts/tree/master/bitnami/common) dependency, which makes it easier to align to [Helm Best Practices](https://helm.sh/docs/chart_best_practices/) by providing things like standardized labels and annotations. The separation of different components into different sub-folders also gives a bit more room to expand in the future, which keeping the layout somewhat manageable.

I like how your PR uses the terminology of "api" and "web" to represent the two components, whereas mine uses "marquez" and "web". I also like how you separated the ingress resource to apply separately to the web and api components and left the configurations flexible. My ingress approach was made a bit more restrictive based on limitations I found within the web component (it seems to only work when running on the same host as the api), but I don't think we should force those constraints into the Helm chart.

As for next steps, how would you feel about the following?

1. Merge my PR #1019 into the main branch first, since it introduces the Bitnami common functionality and contains additional folder structuring changes.
2. Apply your additional changes (noted above, and whatever else you feel is relevant) to a new branch, and make a PR from there. I'm happy to help with this piece to ease the burden of needing to deal with conflict issues (just let me know how I can help).
3. Merge the new PR created in step 2, and live a prosperous life.

**Comment by henneberger** [comment]
These charts look great! The only blocker is to sign off the commit for DCO plugin to pass. Thanks!

**Comment by wslulciuc** [comment]
Superseded by https://github.com/MarquezProject/marquez/pull/1019

**Pull request closed without merging**